code/common/datadog.q

```diff
@@ -29,6 +39,20 @@ isok:{$[.proc.proctype in key .dg.handlers;.dg.handlers .proc.proctype;1b]}
     .lg.w[`sendmetric;"Currently only linux operating systems are supported to send metrics"]]
   }
 
+//define sendmetric and sendevent functions using web request
+.dg.sendevent_webreq:{[event_title;event_text;tags;alert_type]
+  url:`$(dogstatsd_url,"events?api_key=",dogstatsd_apikey);
+  .Q.hp[url;.h.ty`json]
+  .j.j (`title;`text;`priority;`tags;`alert_type)!(event_title;event_text;"normal";$[0h=type tags;","sv tags;tags];alert_type)
+  }
+
+.dg.sendmetric_webreq:{[metric_name;metric_value]
+  url:`$(dogstatsd_url,"series?api_key=",dogstatsd_apikey);
+  unix_time:floor((`long$.z.p)-`long$1970.01.01D00:00)%1e9;
```

Comment:
Don't much efficiency gains here but you alter this so casting needs to occur only once:
```suggestion
  unix_time:floor(`long$.z.p-1970.01.01D00:00)%1e9;
```